Makefile

```diff
@@ -289,7 +289,8 @@ $(TESTS_TARBALL): $(TESTS_MANIFEST)
 	cp $(STAMPFILE) $(ROOT)/tests.buildstamp
 	pfexec ./tools/builder/builder $(TESTS_MANIFEST) $(TESTS_PROTO) \
 	    $(PROTO) $(ROOT)
-	pfexec gtar -C $(TESTS_PROTO) -I pigz -cf $@ .
+	pfexec gtar -C $(TESTS_PROTO) -I pigz -cf $@ $(shell cd $(TESTS_PROTO) \
+		&& ls)
```

Comment:
In theory, if this tarball contained top level files or directories whose names started with `.` then those files would now be excluded. Am I correct? As long as it is never supposed to do that, I'm fine with this change.

Reply:
Made it explicit in the comments now.